l10n_it_ddt/models/stock_picking_package_preparation.py

```diff
@@ -154,6 +154,31 @@ def _compute_display_name(self):
                                                  date=self.date)
         self.display_name = name
 
+    @api.multi
+    def create_invoice(self):
+        # ----- Check if sale order related to ddt are invoiced. Show them.
+        invoiced_sale = [
+            picking.sale_id.id
+            for picking in self.picking_ids
+            if picking.sale_id and picking.sale_id.invoice_ids]
+        if invoiced_sale:
+            return {
+                'type': 'ir.actions.act_window',
+                'view_type': 'form',
+                'view_mode': 'tree,form',
+                'res_model': 'sale.order',
+                'target': 'current',
+                'domain': '[("id", "in", {ids})]'.format(ids=invoiced_sale),
+                }
```

Comment:
No. This is returned when an order as an invoice yet. In this way when you start to invoicing from order, you continue to invoicing from order.